app/steps/ui/documents/template.html

```diff
@@ -81,7 +16,7 @@
       <h2 class="govuk-heading-m">{{ content.heading1 | safe }}</h2>
       <p class="govuk-body">{{ content.text1 | safe }}</p>
 
-      {{ unorderedList(checklistItems) }}
+      {{ unorderedList(fields.checkListItems.value) }}
```

Comment:
don't we need this as ordered so that it is consistently in same order?

Reply:
I believe the unorderedList widget just sticks whatevers passed in, in an <ul> with bullet points. If we stick it in an <ol> then the list will be numbered which I don't think is what we want. You can alter the  moving around the if statements in the documents page util